fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/ExportCommand.java

```diff
@@ -49,35 +100,248 @@ public class ExportCommand extends Command implements ForwardWithSync {
     public ExportCommand(List<String> nameParts, List<String> partitions, String whereSql, String path,
             Map<String, String> fileProperties, BrokerDesc brokerDesc) {
         super(PlanType.EXPORT_COMMAND);
-        this.nameParts = nameParts;
+        this.nameParts = ImmutableList.copyOf(Objects.requireNonNull(nameParts, "nameParts should not be null"));
```

Comment:
all non-primitive params should use `Objects.requireNonNull` to check not null param